deployments/data8/config/common.yaml

```diff
@@ -5,6 +5,11 @@ nfsPVC:
 
 jupyterhub:
   custom:
+    group_profiles:
```

Comment:
the `custom:` key was duplicated, so i moved the contents of the lower block up here.